kas/anchor.go

```diff
@@ -66,6 +68,76 @@ func NewKASAnchor(kasConfig *KASConfig, db AnchorDB, bc BlockChain) *Anchor {
 	}
 }
 
+// AnchorPeriodicBlock periodically anchor blocks to KAS.
+// if given block is invalid, it does nothing.
+func (anchor *Anchor) AnchorPeriodicBlock(block *types.Block) {
+	if !anchor.kasConfig.Anchor {
+		return
+	}
+
+	if block == nil {
+		logger.Error("KAS Anchor : can not anchor nil block")
+		return
+	}
+
+	if block.NumberU64()%anchor.kasConfig.AnchorPeriod != 0 {
+		return
+	}
+
+	anchor.AnchorBlock(block)
```

Comment:
If the returned error won't be handled, `AnchorBlock` should not return an error.

Reply:
This case doesn't not need to handle. But anchor API will handle the error.

Reply:
If the error from `AnchorPeriodicBlock` does not need to be handled, it is not natural that an error is logged inside `AnchorBlock`. How about moving out log outside `AnchorBlock` and leave the log `KASAnchor` in the API?

Reply:
I think they are different between not need to be handled and need to be logged.
Both `AnchorPeriodicBlock` and `API` need to log the error.
But for now `AnchorPeriodicBlock` doesn't need to be handled.

Reply:
The point I want to emphasize is that if you don't need to handle the error, that is not an error. 
As you don't handle the error from `AnchorPeriodicBlock`, for me, it should be a warning, not an error.

Reply:
OK. I will change the caller chose log or not.

Reply:
@ehnuje PTAL. I reflected. :)

Reply:
17be7acc984ed74970560ce1895972f97c3c0da7